### PR TITLE
adding for prediction on HPG

### PR DESCRIPTION
**Pull request by fafsari**
DON'T MERGE
Please review the prediction code for HPG.

### CODE REVIEW

File: multic/segmentationschool/slurm_predicting.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+CODESDIR=/blue/pinaki.sarder/f.afsari/Codes/MultiC_test/multic/segmentationschool
+SIFDIR=/orange/pinaki.sarder/shared_singularity_files
+
+DATADIR=$CODESDIR/test_data
+MODELDIR=$CODESDIR/pretrained_model
+
+CONTAINER=$SIFDIR/multic_segment.sif
+CUDA_LAUNCH_BLOCKING=1
+singularity exec --nv -B $(pwd):/exec/,$DATADIR/:/data/,$MODELDIR/:/model/ $CONTAINER python3 /exec/segmentation_school.py --option predict --base_dir $CODESDIR --modelfile /model/model_final.pth --files /data --wsi_ext svs
```
Comment (sayatmimar) [inline multic/segmentationschool/slurm_predicting.sh:36]: Why we added `wsi_ext` parameter here?

**Review by sayatmimar — CHANGES_REQUESTED** [review]
LGTM! Please add a short description about how to run the slurm script. E.g. what files/folders (for example `slurm_log.out` is needed, user name should be specified in the sh file) should be created by the user, where to place image to predict, model file etc.

**Comment by fafsari** [comment]
> LGTM! Please add a short description about how to run the slurm script. E.g. what files/folders (for example `slurm_log.out` is needed, user name should be specified in the sh file) should be created by the user, where to place image to predict, model file etc.

File: multic/segmentationschool/slurm_predicting.sh (hunk 1 above)
Comment (fafsari) [reply multic/segmentationschool/slurm_predicting.sh:36]: Actually, wsi_ext is not used if the WSI filetypes are svs, scn or ndpi.

**Pull request closed without merging**